database/misc/couchdb_user

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2015, Maykel Moya <mmoya@mmoya.org>
+# Sponsored by ShuttleCloud http://shuttlecloud.com/
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: couchdb_user
+short_description: Add or remove CouchDB users.
+description:
+   - Add or remove CouchDB users from a remote host.
+version_added: "0.x"
+options:
+  name:
+    description:
+      - name of the user to add or remove
+    required: true
+    default: null
+  password:
+    description:
+      - password of the user
+  roles:
+    description
+      - list of roles of the user
+  login_user:
+    description:
+      - The username used to authenticate to CouchDB
+    required: false
+    default: null
+  login_password:
+    description:
+      - The password used to authenticate to CouchDB
+    required: false
+    default: null
+  login_host:
+    description:
+      - Host of the CouchDB server
+    required: false
+    default: localhost
+  login_port:
+    description:
+      - Port of the CouchDB server
+    required: false
+    default: 5984
+  state:
+    description:
+      - The user state
+    required: false
+    default: present
+    choices: [ "present", "absent" ]
+notes:
+   - Requires the requests python library on the remote host. In Debian systems
+     you can apt-get install python-requests (see M(apt)). In RedHat systems
+     you can yum install python-requests (see M(yum)).
+   - Requires the passlib python library on the remote host. In Debian systems
+     you can apt-get install python-passlib.
+   - Both I(login_user) and I(login_password) are required if you pass one of
+     them.
+requirements: [ passlib, requests ]
+author: Maykel Moya
+'''
+
+
+EXAMPLES = '''
+# Create a new user with name 'bob'
+- couchdb_user: name=bob password=bar state=present
+
+'''
+
+
+from binascii import hexlify
+from passlib.utils.pbkdf2 import pbkdf2
+import requests
```

Comment:
There is a `fetch_url` helper for interaction with URL which doesn't depend on request (also for other methods as GET). https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/urls.py. 

If this is an option for you, since users may don't have requests installed.